packages/sdk/src/Sygma.ts

```diff
@@ -1015,4 +1018,75 @@ export class Sygma implements SygmaSDK {
   public getDestinationChainProvider(): ethers.providers.JsonRpcProvider {
     return this.providers!.chain2 as ethers.providers.JsonRpcProvider;
   }
+
+  /**
+   * @name getFeeRouterAddress
+   * @param chain - the chain to get the fee router address
+   * @returns {string} - the address of the fee router
+   */
+  public getFeeRouterAddress(chain: 'chain1' | 'chain2'): string {
```

Comment:
There is a type for `chain1' | 'chain2'`, called [`Directions`](https://github.com/sygmaprotocol/sygma-sdk/blob/47df66e41314cd225165fabe50167f96ca80c8b5/packages/sdk/src/types/types.ts#L8). 

Even though I am not sure if that is the best name for this type, let us use the defined type everywhere - it is easy to rename it in the future.